.github/workflows/test.yml

```
@@ -279,28 +283,58 @@
-        run: chmod +x /usr/local/bin/atmos
+      - name: Add build artifacts directory to PATH for ${{ matrix.flavor.target }}
+        if: matrix.flavor.target == 'linux' || matrix.flavor.target == 'macos'
+        run: |
+          echo "${{ github.workspace }}" >> $GITHUB_PATH
+          chmod +x "${{ github.workspace }}/atmos
 
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v4
+      - name: Add build artifacts directory to PATH for ${{ matrix.flavor.target }}
+        if: matrix.flavor.target == 'windows'
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:35: Double quote to prevent globbing and word splitting

(shellcheck)